vagrant/Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.synced_folder "../", "/home/vagrant/sources"
```

Comment:
(to discuss) wouldn't it be better if repo directory will be mounted under `/home/vagrant/src/ralph` (notice sources -> src and additional ralph directory)